### ISSUE TITLE
DyNerf datasets and ENeRF-Outdoor datasets config 

### ISSUE THREAD

**Issue by ch1998** [zju3dv/im4d#7]
Excellent work. I saw the results of DyNerf and outdoor datasets on the project page, but I did not find the relevant config and data processing files in the project. Will they be added in the future?

**Comment by haotongl** [comment]
Yes, if everything goes well, I will release this part of the code before SIGGRAPH Asia 2023

**Comment by ch1998** [comment]
I used the enerf-outsider data set, re-colmap, and used the conversion code in easymocap to get intri/extri.yml.
Tested on actor5, and the exported mesh results are as shown below.
But I got the following error while rendering the image.

mesh
![image](https://github.com/zju3dv/im4d/assets/37291486/e5b95914-cd0c-476c-8d49-326be45abd13)

error
![image](https://github.com/zju3dv/im4d/assets/37291486/0bbde086-6f13-44d8-bf1e-965fc290ae85)

Is it an error caused by camera parameters?
